docs/quick-start-guide.md

```diff
@@ -38,6 +40,44 @@ Now you should see the operator running in the cluster by checking the status of
+
+The operator uses multiple workers in the `SparkApplication` controller and the submission runner.
+The number of worker threads to use in the three places are controlled using command-line flags `-controller-threads` and `-submission-threads`, respectively. The default values for the flags are 10 and 3, respectively.
+
+The operator enables cache resynchronization so periodically the informers used by the operator will re-list existing objects it manages and re-trigger resource events. The resynchronization interval in seconds can be configured using the flag `-resync-interval`, with a default value of 30 seconds.
+
+By default, the operator will install the [CustomResourceDefinitions](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/) for the custom resources it managers. This can be disabled by setting the flag `-install-crds=false`.
+
+The mutating admission webhook is an **optional** component and can be enabled or disabled using the `-enable-webhook` flag, which defaults to `false`.
+
+By default, the operator will manage custom resource objects of the managed CRD types for the whole cluster. It can be configured to manage only the custom resource objects in a specific namespace with the flag `-namespace=<namespace>`
+
+## Using the Mutating Admission Webhook
+
+The Kubernetes Operator for "Apache Spark comes with an optional mutating admission webhook for customizing Spark driver and executor pods based on the specification in `SparkApplication` objects, e.g., mounting user-specified ConfigMaps and volumes, and setting pod affinity/anti-affinity.
+
+The webhook requires a X509 certificate for TLS for pod admission requests and responses between the Kubernetes API server and the webhook server running inside the operator. For that, the certificate and key files must be accessible by the webhook server.
+The Spark Operator ships with a tool at `hack/gencerts.sh` for generating the CA and server certificate and putting the certificate and key files into a secret named `spark-webhook-certs` in namespace `sparkoperator`. This secret will be mounted into the Spark Operator pod.  
+
+Run the following command to create secret with certificate and key files using Batch Job, and install the Spark Operator Deployment with the mutating admission webhook:
+
+```bash
+$ kubectl apply -f manifest/spark-operator-with-webhook.yaml
+```
+
+This will create a Deployment named `sparkoperator` and a Service named `spark-webhook` for the webhook in namespace `sparkoperator`.
+
+If the operator is installed via the Helm chart using the default settings (i.e. with webhook enabled), the above steps are all automated for you.
+
+### Default Spark Image
+
+Operator allows to set default unified spark image for driver and executors, using `-default-spark-image=[image]:[tag]` flag. Each submitted spark application will use that image as default, upon customized by the user in the `SparkApplication` template.
```

Comment:
`Operator` -> `The operator`. Also please use `Spark` instead of `spark` consistently.